recipes/trimal/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
   name: trimal
-  version: "1.4.1"
+  version: "1.5.0"
 
 source:
-  url: https://github.com/scapella/trimal/archive/v1.4.1.tar.gz
-  md5: cb193397635da45be350e9ac96dafe45
+  url: https://github.com/inab/trimal/archive/v1.5.0.tar.gz
```

Comment:
I see that Github automatically from the old repo to the new one, so I guess it is OK!

Reply:
Hi @martin-g - It was a decision I made to guarantee the long-term sustainability of the code so it is maintained at an organisational level rather than through a personal repository. At the moment, there is no real difference as I'm also responsible for the organisation.

Reply:
Thanks for the confirmation, @scapella !